games-action/supertuxkart/supertuxkart-1.0.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils desktop gnome2-utils
```

Comment:
If you drop gnome2-utils and switch to xdg-utils, you should be able to bump EAPI to 7 in this ebuild.